src/eos-updater-fetch.c

```diff
@@ -964,7 +1071,7 @@ content_fetch (FetchData     *fetch_data,
       if (data->results == NULL)
         g_message ("Fetch: using old code due to lack of repo finder results");
 
-      if (content_fetch_old (fetch_data, context, cancellable, &local_error))
+      if (content_fetch_old (fetch_data, context, fetch_cancellable, &local_error))
```

Comment:
I suspect you also need to use `fetch_cancellable` in the `prepare_flatpaks_to_deploy()` call below.

Reply:
True! For some reason I thought the `prepare_flatpaks_to_deploy` function was not downloading stuff but I've verified that's not the case.